airflow-core/src/airflow/api_fastapi/core_api/datamodels/assets.py

```diff
@@ -160,6 +160,20 @@ class QueuedEventCollectionResponse(BaseModel):
     total_entries: int
 
 
+class CreateAssetBody(StrictBaseModel):
+    """Create asset request."""
+
+    name: str
+    uri: str
+    group: str
+    extra: dict = Field(default_factory=dict)
+
+    class Config:
+        """Pydantic config."""
+
+        extra = "forbid"
```

Comment:
It would be better to follow the Pydantic V2 syntax.
```suggestion
    model_config = ConfigDict(extra="forbid")
```